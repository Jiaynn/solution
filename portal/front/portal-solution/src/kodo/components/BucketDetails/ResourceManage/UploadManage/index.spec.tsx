/**
 * @file test cases for component UploadManage
 * @author zhangheng01 <zhangheng01@qiniu.com>
 */

// jest.mock('kodo/stores/config')

// import * as React from 'react'
// import { renderer } from 'portal-base/common/utils/test'

// import { mockAntdComponents } from 'kodo/test/utils'

// import configStore from 'kodo/stores/config'
// import Store from '../store'

// configStore.init()
// mockAntdComponents()
// import Upload from './index'

// it('renders correctly', () => {
//   const props = {
//     bucketName: 'test',
//     region: 'z1',
//     domain: 'qiniu.com',
//     isPrivate: false,
//     versionEnabled: true
//   }
//   const store = new Store('test')
//   const tree = renderer.create(
//     <Upload {...props} store={store} />
//   ).toJSON()
//   expect(tree).toMatchSnapshot()
// })
