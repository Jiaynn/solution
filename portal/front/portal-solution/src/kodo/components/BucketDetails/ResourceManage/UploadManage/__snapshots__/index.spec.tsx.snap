// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="ant-row"
  >
    <div
      className="ant-col ant-col-16"
    >
      <div
        className="ant-card ant-card-bordered"
      >
        <div
          className="ant-card-head"
          style={Object {}}
        >
          <div
            className="ant-card-head-wrapper"
          >
            <div
              className="ant-card-head-title"
            >
              上传文件至 test
            </div>
          </div>
        </div>
        <div
          className="ant-card-body"
          style={Object {}}
        >
          <table>
            <tbody />
          </table>
        </div>
      </div>
      <div
        className="ant-row"
      >
        <div
          className="ant-col ant-col-12"
        >
          <a
            onClick={[Function]}
          >
            返回
          </a>
        </div>
        <div
          className="ant-col ant-col-12"
        >
          <div
            data-sensors-click={true}
            name="sensors-upload-select-files"
          >
            <span>
              选择文件
            </span>
            <input
              disabled={true}
              multiple={true}
              onChange={[Function]}
              type="file"
            />
          </div>
        </div>
      </div>
    </div>
    <div
      className="ant-col ant-col-7 ant-col-offset-1"
    >
      <div>
        <div>
          <p>
            选择上传文件存储类型
          </p>
          <div
            className="ant-select ant-select-enabled"
            onBlur={[Function]}
            onClick={[Function]}
            onFocus={[Function]}
            onMouseDown={[Function]}
            onMouseOut={[Function]}
            onMouseUp={[Function]}
            onTouchStart={[Function]}
            style={
              Object {
                "width": "100%",
              }
            }
          >
            <div
              aria-autocomplete="list"
              aria-controls="test-uuid"
              aria-expanded={false}
              aria-haspopup="true"
              className="ant-select-selection
            ant-select-selection--single"
              onKeyDown={[Function]}
              role="combobox"
              tabIndex={0}
            >
              <div
                className="ant-select-selection__rendered"
              >
                <div
                  className="ant-select-selection__placeholder"
                  onClick={[Function]}
                  onMouseDown={[Function]}
                  style={
                    Object {
                      "WebkitUserSelect": "none",
                      "display": "none",
                      "userSelect": "none",
                    }
                  }
                  unselectable="on"
                >
                  请选择存储类型
                </div>
                <div
                  className="ant-select-selection-selected-value"
                  style={
                    Object {
                      "display": "block",
                      "opacity": 1,
                    }
                  }
                  title="标准存储"
                >
                  标准存储
                </div>
              </div>
              <span
                className="ant-select-arrow"
                onClick={[Function]}
                style={
                  Object {
                    "WebkitUserSelect": "none",
                    "userSelect": "none",
                  }
                }
                unselectable="on"
              >
                <i
                  aria-label="icon: down"
                  className="anticon anticon-down ant-select-arrow-icon"
                >
                  <svg
                    aria-hidden="true"
                    className=""
                    data-icon="down"
                    fill="currentColor"
                    focusable="false"
                    height="1em"
                    viewBox="64 64 896 896"
                    width="1em"
                  >
                    <path
                      d="M884 256h-75c-5.1 0-9.9 2.5-12.9 6.6L512 654.2 227.9 262.6c-3-4.1-7.8-6.6-12.9-6.6h-75c-6.5 0-10.3 7.4-6.5 12.7l352.6 486.1c12.8 17.6 39 17.6 51.7 0l352.6-486.1c3.9-5.3.1-12.7-6.4-12.7z"
                    />
                  </svg>
                </i>
              </span>
            </div>
          </div>
        </div>
        <div>
          <p>
            设置路径前缀：
          </p>
          <input
            className="ant-input"
            disabled={false}
            onChange={[Function]}
            onKeyDown={[Function]}
            placeholder="示例：image/**"
            type="text"
            value=""
          />
        </div>
        <div>
          <i
            aria-label="icon: question-circle"
            className="anticon anticon-question-circle"
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
          >
            <svg
              aria-hidden="true"
              className=""
              data-icon="question-circle"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm0 820c-205.4 0-372-166.6-372-372s166.6-372 372-372 372 166.6 372 372-166.6 372-372 372z"
              />
              <path
                d="M623.6 316.7C593.6 290.4 554 276 512 276s-81.6 14.5-111.6 40.7C369.2 344 352 380.7 352 420v7.6c0 4.4 3.6 8 8 8h48c4.4 0 8-3.6 8-8V420c0-44.1 43.1-80 96-80s96 35.9 96 80c0 31.1-22 59.6-56.1 72.7-21.2 8.1-39.2 22.3-52.1 40.9-13.1 19-19.9 41.8-19.9 64.9V620c0 4.4 3.6 8 8 8h48c4.4 0 8-3.6 8-8v-22.7a48.3 48.3 0 0 1 30.9-44.8c59-22.7 97.1-74.7 97.1-132.5.1-39.3-17.1-76-48.3-103.3zM472 732a40 40 0 1 0 80 0 40 40 0 1 0-80 0z"
              />
            </svg>
          </i>
          <span>
            上传覆盖：
          </span>
          <button
            aria-checked={false}
            checked={false}
            className="ant-switch"
            defaultChecked={false}
            onChange={[Function]}
            onClick={[Function]}
            onKeyDown={[Function]}
            onMouseUp={[Function]}
            role="switch"
            type="button"
          >
            <span
              className="ant-switch-inner"
            >
              关闭
            </span>
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
